starter-code/index.js

```diff
@@ -1,17 +1,46 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
```

Comment:
Aqui você já atualizou a propriedade `length` (linha 4) com o tamanho do array `items,` assim, você poderia também usar nas linhas 21, 28 e 39 a propriedade `this.length` ao invés de `this.items.length`.

Do jeito que você fez não está errado, estou apenas sugerindo mais uma abordagem. :)